### PR TITLE
feat: Add domain 5 for rococo phala on devnet

### PR DESCRIPTION
**Pull request by freddyli7**
- Add Rococo phala to devnet config file

### CODE REVIEW

File: config.test.json [hunk 1]
```diff
@@ -176,6 +176,39 @@
...
       ]
+    },
+    {
+      "id": 4,
+      "name": "rococo-phala",
+      "type": "substrate",
+      "bridge": "",
+      "handlers": [
+        {
+          "type": "",
```
Comment (tcar121293) [inline config.test.json:187]: I think you can just leave the handlers array empty like `[]`

**Review by tcar121293 — APPROVED** [review]

**Review by mpetrun5 — CHANGES_REQUESTED** [review]
I think sepolia is already deployed as id of 4

**Comment by nmlinaric** [comment]
Actually this is my mistake, I didn't know `domainIDs˙ on testnet start from 1, not from 0 like on devnet. The question now is, is this already deployed on rococo? If yes, I'll redeploy to Sepolia with `domainID`=5, if not bridge should be deployed to rococo with `domainID`=5.
cc: @freddyli7

**Review by nmlinaric — APPROVED** [review]

File: config.dev.json [hunk 2]
```diff
@@ -235,6 +235,34 @@
...
+      "bridge": "",
+      "handlers": [],
+      "nativeTokenSymbol": "pha",
+      "nativeTokenFullName": "pha",
+      "nativeTokenDecimals": 18,
+      "blockConfirmations": 2,
+      "startBlock": 0,
+      "resources": [
+        {
+          "resourceId": "0x0000000000000000000000000000000000000000000000000000000000000001",
```
Comment (nmlinaric) [inline config.dev.json:252]: This also should be `0x000...1000`

**Review by nmlinaric — CHANGES_REQUESTED** [review]

File: config.dev.json [hunk 3]
```diff
@@ -235,6 +235,34 @@
...
+          "type": "erc20",
+          "address": "",
+          "symbol": "PHA",
+          "decimals": 18
```
Comment (mpetrun5) [inline config.dev.json:256]: Are the decimals right?

**Review by mpetrun5 — CHANGES_REQUESTED** [review]

File: config.dev.json (hunk 3 above)
Comment (freddyli7) [reply config.dev.json:256]: this is the erc20 token decimal, so should be 18, on devnet, there is no official PHA token but I think that we could use is a Lock/Release erc20 token just pretend to be PHA which has 18 decimals

**Comment by freddyli7** [comment]
@tcar121293 @nmlinaric @mpetrun5 guys, after sync with @P1sar and Phala, now there is only one public test env on phala so we can only connect Rococo to our devnet, so I removed the changes in config.test.json file. please review

File: config.dev.json [hunk 4]
```diff
@@ -235,6 +235,34 @@
...
+      "id": 5,
+      "name": "rococo-phala",
+      "type": "substrate",
+      "bridge": "",
+      "handlers": [],
+      "nativeTokenSymbol": "pha",
+      "nativeTokenFullName": "pha",
+      "nativeTokenDecimals": 18,
+      "blockConfirmations": 2,
+      "startBlock": 0,
```
Comment (mpetrun5) [inline config.dev.json:249]: Should this be 0?
We should probably set this to the latest block currently on rococo as we don't want to index from the beginning of time.

**Review by mpetrun5 — CHANGES_REQUESTED** [review]

File: config.dev.json (hunk 4 above)
Comment (freddyli7) [reply config.dev.json:249]: that makes sense, will do

Comment (freddyli7) [reply config.dev.json]: done, please check

**Review by mpetrun5 — APPROVED** [review]

File: config.dev.json (hunk 4 above)
Comment (mpetrun5) [reply config.dev.json]: Is a 0 missing?
Seems low.

File: config.dev.json [hunk 5]
```diff
@@ -235,6 +235,34 @@
...
+          "type": "erc20",
+          "address": "",
+          "symbol": "PHA",
+          "decimals": 18
+        },
+        {
+          "resourceId": "0x0000000000000000000000000000000000000000000000000000000000000300",
+          "type": "erc20",
+          "address": "",
+          "symbol": "USDC",
```
[the remark this answers is not in the document]
Comment (freddyli7) [reply config.dev.json]: updated the resourceID to `0x000000000000000000000000000000000000000000000000000000000000000` and token as ERC20TST

**Review by tcar121293 — APPROVED** [review]

**Review by nmlinaric — APPROVED** [review]

**Pull request merged**